lhotse/cut.py

```diff
@@ -102,13 +114,21 @@ def load_audio(self, recording_set: RecordingSet, root_dir: Optional[Pathlike] =
         :param root_dir: optional Path prefix to find the recording in the filesystem.
         :return: a numpy ndarray with audio samples, with shape (1 <channel>, N <samples>)
         """
-        return recording_set.load_audio(
-            self.recording_id,
-            channels=self.channel,
-            offset_seconds=self.start,
-            duration_seconds=self.duration,
-            root_dir=root_dir
-        )
+        if recording_set is None:
```

Comment:
Can we remove the `recording_set` argument? At this point, if a user wants to be able to load audio from the Cut, they should just attach the recording to it. It will simplify this function.